hardcoded.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+// Used to generate data for consts.php INSTALLER_TO_REPO_MINOR_VERSIONS
+// Copy paste in .cow.pat.json from the current release (file is .gitignored)
+
+include 'consts.php';
+
+$fn = '.cow.pat.json';
+if (!file_exists($fn)) {
+    echo "Create $fn\n";
```

Comment:
`$fn` is normally shorthand for "function". If it was just used here I'd probably be okay with it, but since it's used below we should be more explicit with the name.
```suggestion
$fileName = '.cow.pat.json';
if (!file_exists($fileName)) {
    echo "Create $fileName\n";
```

Reply:
updated